includes/gateways/stripe/includes/payment-methods/class-give-stripe-checkout.php

```diff
@@ -63,6 +65,38 @@ public function __construct() {
 
 		}
 
+		public function output_redirect_notice() {
```

Comment:
@henryholtgeerts This function does not have a description.

Reply:
fixed: https://github.com/impress-org/givewp/commit/aba541debb24b5eb9855d9daaa292b8e1ec6096c

Reply:
@henryholtgeerts what is use of this function? This function functionality seems Sequoia form template specific only